extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -11,16 +11,19 @@ import { IDataComponent } from '../../interfaces';
 import { PredictColumn, DatabaseTable, TableColumn } from '../../../prediction/interfaces';
 import { ModelParameter, ModelParameters } from '../../../modelManagement/interfaces';
 
+export const _warningButtonDimensions = {
```

Comment:
I think export is not needed here, this is only to be used in this file, also a const should be named to WarningButtonDimension, _xxx is for private members in a class.

Reply:
Will correct this.